docs-kits/kits/Traceability Kit/Software Development View/page_app-provider_software-development-view.mdx

```diff
@@ -46,7 +47,32 @@ The notification states and their cycle are described in the following figure.
+
+In this case, block notifications include a **notification status similar to quality notifications** to track communication, but **also include a new status model** for each individual part of the block notification to provide additional information. For example, whether the part was blocked or whether the process to block a part was canceled.
+
+> :raised_hand: Since the Catena-x unique ID is used for the individually listed (damaged) parts in the block notification, the app provider should also provide functions for the standardized creation of digital twins of vehicles, parts and materials, as already mentioned in the quality notifications section. 
+
+### Block Status Model
+In order to track the blocking process in the Catena X network, a defined status model is used for each part of the block notification:
+![Block Status Model](../assets/block-notification-state-model.svg)
+
+- ACTIVE means that the part has been identified as a damaged and safety-critical part and must therefore be blocked on the customer side.
+- PART_BLOCKED is used when the recipient has received the block notification and actually blocks / sorts out the damaged parts as a measure.
+- CANCELED serves as the update status of the component originally identified to be blocked if the supplier subsequently determines that the original part does meet safety requirements, is not damaged or that the information was sent by mistake. This status can also be set by the manufacturer if, after an (initial) analysis, the part does not require a block.
+
+### Block Notification API
+A standardized API and corresponding payloads are specified for block notification to enable and ensure the exchange of information that is critical to the blocking process in a standardized way. At this point, the notification API is focused on sending and receiving notifications with a full stack of block information and on updating a previously sent notification by changing the block status. The implemented endpoints shall be able to process the defined request body and respond with the HTTP status codes and - if required - reply with the defined response body. Meanwhile, the Block Notification API shall only be accessible after successful contract negotiation via Connector based on [Dataspace Protocol (DSP)](https://docs.internationaldataspaces.org/dataspace-protocol/), since the API is made available as part of an EDC data asset with usage policy attached. Please refer to the corresponding Block Notification API specification for more details:
+- [Block Notification API (v1.0.0)](../openapi/block-notifications-1-0-0.yaml)
+
+> :raised_hand: For  the current release, **version 1.0.0 is optional** and MUST be supported by every App provider from the next (major) release. 
```

Comment:
I think we should be transparent here and mention, that Block notifications are currently (R24.12) not standardized yet. Additionally, it could be mentioned what / when the next major release will be published - R25.09